src/main/java/frc/robot/poseestimator/helpers/objectListOrganizer.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.poseestimator.helpers;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Supplier;
+
+public class objectListOrganizer<T> {
```

Comment:
[Java naming conventions](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html) call for using [camel case](https://en.wikipedia.org/wiki/Camel_case) for class names.
This name does not reflect the true purpose of this class, replace it